ballerina/xmldata.bal

```diff
@@ -149,8 +166,8 @@ isolated function getElement(string name, xml children, map<string> attributes =
     return element;
 }
 
-isolated function getAttributesMap(json jTree, map<string> parentNamespaces = {}, JsonOptions options = {})
-                            returns map<string>|Error {
+isolated function getAttributesMap(json jTree, map<string> namespaces, map<string> parentNamespaces = {},
+                                    JsonOptions options = {}) returns map<string>|Error {
     string attributePrefix = options.attributePrefix == "" ? "@" : options.attributePrefix;
     map<string> attributes = parentNamespaces.clone();
     map<json>|error attr = jTree.ensureType();
```

Comment:
Don't we need to handle this error? seems like we are ignoring

Reply:
Yes, we have to skip this error. 

This is a recursive function and it is used to get attribute values. If `jTree` can be converted to a `map<json>`, we have to get the attribute values from that input. Otherwise, we have to skip this.